R/tm_missing_data.R

```diff
@@ -381,25 +409,30 @@ encoding_missing_data <- function(id, summary_per_patient = FALSE, ggtheme, data
           ),
           value = FALSE
         )
-      }
+      },
+      ui_decorate_teal_data(ns("dec_summary_plot"), decorators = subset_decorators("summary_plot", decorators))
```

Comment:
`subset_decorators("summary_plot", decorators)` will return all from `default` plus the `summary_plot` related.

We can use either the names of the objects, or more generic ones